src/react-apps/applications/shared/src/navigation/drawer/drawerMenuSettings.ts

```diff
@@ -0,0 +1,148 @@
+export interface IMenuItem {
+  displayText: string;
+  navLink: string;
+  menuType?: string;
```

Comment:
Why is `menuType` optional? It's on every items i've seen in the declaration of `mainMenuSettings` and `leftDrawerMenuSettings`.

Reply:
This interface is shared between main menu and drawer menu. Menu type is used only in main menu. So i set that as optional